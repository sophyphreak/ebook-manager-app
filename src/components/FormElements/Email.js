import React, { Component } from 'react';
import BasicInput from './MainElements/BasicInput';

const Email = ({
  label,
  email,
  onEmailChange,
  ...props
}) => (
  <BasicInput
    label={label}
    value={email}
    onChange={onEmailChange}
    {...props}
  />
);

export default Email;