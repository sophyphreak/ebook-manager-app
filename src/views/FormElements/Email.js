import React, { Component } from 'react';

const Email = (props) => (
  <label>
    Email:*
    <input type="text" value={props.value} onChange={props.onChange} />
  </label>
);

export default Email;