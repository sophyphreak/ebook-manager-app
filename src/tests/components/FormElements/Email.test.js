import React from 'react';
import { shallow } from 'enzyme';
import Email from '../../../components/FormElements/Email';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Email />);
});

test('should render Email correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
