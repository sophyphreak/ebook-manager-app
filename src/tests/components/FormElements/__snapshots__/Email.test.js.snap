// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Email correctly 1`] = `<BasicInput />`;
